scripts/PearlPowderISIS/pearl_cycle_factory.py

```diff
@@ -0,0 +1,154 @@
+# pylint: disable=anomalous-backslash-in-string, too-many-branches, superfluous-parens
+
+from mantid.simpleapi import *
+import pearl_routines
+
+
+def get_cycle_dir(number, currentdatadir):
+    if type(number) is int:
+        runno = number
+    else:
+        num = number.split("_")
+        runno = int(num[0])
+
+    print "The run number is get cycle is", runno
+    if (runno < 71009):
+        cycle = "10_2"
+        print "cycle is set to", cycle
+        instver = "old"
+        datadir = "X:\data\cycle_" + cycle + "\\"
```

Comment:
Ah I see. I'll use that trick to see if it all can run without network. Let's note that down to check with Craig if the usually/ever use the `pearl_datadir` and `currentdatadir`.